src/drivers/net/e1000.c

```diff
@@ -284,7 +291,7 @@ static int e1000_open(struct net_device *dev) {
 
 	for (size_t i = 0; i < E1000_RXDESC_NR; i ++) {
 	        struct sk_buff *skb;
-		skb = skb_alloc(ETH_FRAME_LEN);
+		skb = skb_alloc(E1000_MAX_RX_LEN);
 		if (skb == NULL)
 			panic("Can't allocate skb %d", i);
```

Comment:
And this one should probably refuse to open the device, i.e. throw ENOMEM maybe?